velox/exec/Operator.h

```diff
@@ -761,3 +761,12 @@ class SourceOperator : public Operator {
   }
 };
 } // namespace facebook::velox::exec
+
+template <>
+struct fmt::formatter<std::thread::id> : formatter<std::string> {
+  auto format(std::thread::id s, format_context& ctx) {
+    std::ostringstream oss;
+    oss << s;
+    return formatter<std::string>::format(oss.str(), ctx);
+  }
+};
```

Comment:
This might now be covered by `fmt/std.h` https://github.com/wangzw/fmt/blob/e57ca2e3685b160617d3d95fcd9e789c4e06ca88/include/fmt/std.h#L146-L150

Reply:
Ah nice, I'll leave it as is for now until we know if we want to support <9 or not.